lib/base/journaldlogger.cpp

```diff
@@ -0,0 +1,60 @@
+/* Icinga 2 | (c) 2021 Icinga GmbH | GPLv2+ */
+
+#include "config.h"
+#if !defined(_WIN32) && defined(HAVE_SYSTEMD)
+#include "base/journaldlogger.hpp"
```

Comment:
Same here, either include `i2-base.hpp` or just move `journaldlogger.hpp` above the `#if`.